src/sql/workbench/browser/modelComponents/loadingComponent.component.ts

```diff
@@ -19,7 +19,7 @@ import { ILogService } from 'vs/platform/log/common/log';
 @Component({
 	selector: 'modelview-loadingComponent',
 	template: `
-		<div class="modelview-loadingComponent-container" aria-busy="true" *ngIf="loading">
+		<div class="modelview-loadingComponent-container" aria-busy="true" *ngIf="loading" [ngStyle]="CSSStyles">
```

Comment:
Are there any scenarios that we don't want it to be vertically center aligned? maybe we should just add the style directly here. can you try other ways to do the alignment instead of setting the height?

Reply:
@alanrenmsft, using margin-top instead of height to get the same effect.

Reply:
i guess that is ok